File: cms_web/src/Services/Entities/OtpEntity.ts
interface OtpEntity {
  otp_pk?: string;
  user_pk?: string;
  mob_no?: string;
  otp_code?: string;
  expire_sec?: string;
  is_verified?: string;
  consult_req_pk?: string;
  encoded_at?: Date | string;
}

export default OtpEntity;

File: cms_server/cms_server/Entities/OtpEntity.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace cms_server.Entities
{
    public class OtpEntity
    {
        public int otp_pk { get; set; }
        public string user_pk { get; set; }
        public string mob_no { get; set; }
        public string otp_code { get; set; }
        public int expire_sec { get; set; }
        public string is_verified { get; set; }
        public DateTime encoded_at { get; set; }
        public string consult_req_pk { get; set; }
    }
}
